lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -1,7 +1,14 @@
 CFCUlxCommands.constrainChecker = CFCUlxCommands.constrainChecker or {}
 local cmd = CFCUlxCommands.constrainChecker
 local CATEGORY_NAME = "Utility"
-local IsValid = IsValid
+local IsValid, Clamp, Remap, Round = IsValid, math.Clamp, math.Remap, math.Round
+local HSV_RED_ANGLE, HSV_GREEN_ANGLE = 0, 120
+local WHITE = Color( 255, 255, 255 )
+local CONSTRAINT_THRESHOLD = 500
+local NETWORK_NAME = "CFC_ulx-constraint_checker"
+if SERVER then
+    util.AddNetworkString( NETWORK_NAME )
```

Comment:
You can just put the network string directly into this, you dont really need it as a var.

Reply:
👍